libmscore/measure.cpp

```diff
@@ -2588,10 +2596,10 @@ void Measure::scanElements(void* data, void (*func)(void*, Element*), bool all)
 
       int nstaves = score()->nstaves();
       for (int staffIdx = 0; staffIdx < nstaves; ++staffIdx) {
-            if (!all && !(visible(staffIdx) && score()->staff(staffIdx)->show()))
+            if (!all && !score()->staff(staffIdx)->show())
```

Comment:
why did you remove that `!(visible(staffIdx))` check? You left only the score-wise check and removed the measure-wise check.

Reply:
I'll check for potential exceptions here, thank you!.

Reply:
there's other elements than spacers and breaks in this for. Maybe include this check somewhere else for the other elements?

Reply:
Sorry, I meant breaks, spacers, measure numbers and mmRangeText, introduced recently by you, but none seems related to the exception introduced some merges ago.